### ISSUE TITLE
NoTrayIcon doesn't work. Please help

### ISSUE THREAD

**Issue by Iz-zzzzz** [spyoungtech/ahk#211]
This is my code.
```
from ahk import AHK
from pathlib import Path
from ahk.directives import NoTrayIcon

AHK_EXE = Path(__file__).parent / "AutoHotkey.exe"
ahk = AHK(executable_path=str(AHK_EXE), directives=[NoTrayIcon])

def my_callback():
    print('Hello callback!')

# when WIN + n is pressed, fire `my_callback`
ahk.add_hotkey('#n', callback=my_callback)
ahk.start_hotkeys()  # start the hotkey process thread
ahk.block_forever()  # not strictly needed in all scripts -- stops the script from exiting; sleep forever
```
I have already added "directives=[NoTrayIcon]" but the AHK icon still shows.
![image](https://github.com/spyoungtech/ahk/assets/57972212/ea3ea9b3-f4bc-4c9c-9131-e1a176762df5)

AutoHotkey 1.1.36.02
Python 3.11 (Lastest version from MS Store)
Windows 10 Education 19045.3086

**Comment by spyoungtech** [comment]
Right now, the `directives` keyword does not affect the process used for Hotkeys/hotstrings. So, when you use hotkeys, you'll still see the tray icon.

I'll plan on adding functionality to address this use case in an upcoming release.

**Comment by spyoungtech** [comment]
PR #217 addresses this issue and will likely be released in v1.2.0

A release candidate containing this feature is available in `1.2.0rc2` which has been published to PyPI (`pip install "ahk==1.2.0rc2"`)

**Comment by spyoungtech** [comment]
1.2.0 is now released 🎉
